lessons/03_wave/03_03_aBetterModel.ipynb

```diff
@@ -85,23 +85,23 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "A good question to ask here is:  should the maximum flux on a given stretch of road have a strict dependence on the maximum speed that the roadway allows, be it by physical restrictions or posted speed limits?  \n",
+      "A good question to ask here is:  should the maximum flux on a given stretch of road have a strict dependence on the maximum speed that the roadway allows, be it by physical restrictions or posted speed limits? In other words, do we really expect the maximum flux to increase if we allow arbitrarily high speeds? \n",
       "\n",
       "Probably not.  But there *should* be some ideal traffic speed, $u^{\\star}$, corresponding to an ideal traffic density, $\\rho^{\\star}$, resulting in the maximum traffic flux:\n",
       "\n",
       "\\begin{equation}F_{\\rm max} = \\rho^{\\star}u^{\\star}\\end{equation}\n",
       "\n",
-      "Any flux model would still satisfy that the slope of the curve $F(\\rho)$ is equal to zero at $F_{\\rm max}$. One possible flux model that can satisfy all the conditions is cubic in $\\rho$ instead of quadratic: \n",
+      "Let us improve the initial flux model by taking this observation into account. The obvious thing to try is to introduce a flux model that is cubic in $\\rho$ instead of quadratic:\n",
```

Comment:
I would like to not use the phrase "the obvious" anything throughout the course.  If a student doesn't find it obvious it can be quite demoralizing.